client/index.html

```diff
@@ -175,6 +180,7 @@ <h1 class="title">Connect</h1>
 				<div id="settings" class="window">
 					<div class="header">
 						<button class="lt" aria-label="Toggle channel list"></button>
+						<span class="title">Settings</span>
```

Comment:
Meh, I'm not sure it adds value, but we'll see.

In the meantime, my opinion is we should really not have this top titles on the settings and connect pages.